engine/table/src/main/java/io/deephaven/engine/table/impl/UpdateBy.java

```diff
@@ -106,14 +232,19 @@ public static Table updateBy(@NotNull final QueryTable source,
                         throw new IllegalStateException("Unsupported redirection type " + type);
                 }
             }
+        } else {
+            rowRedirection = null;
         }
 
+        // create an UpdateByRedirectionContext for use by the UpdateBy objects
+        UpdateByRedirectionContext ctx = new UpdateByRedirectionContext(rowRedirection);
```

Comment:
The RedirectionContext (now named RedirectionHelper) is passed into the operators for creation of dense vs. sparse output sources. It will still be needed in some form.